src/data/pages/index.en.mdx

```diff
@@ -117,13 +117,13 @@ import FaqSection from "../../components/widgets/faq/Wrapper.astro";
       title: "Development",
       version: "3._",
       variant: "inprogress",
-      body: `Currently we are still in development. You can see the <a class="underline" href="https://github.com/ecamp/ecamp3/milestone/7">current status</a> on GitHub.`,
+      body: `Currently we are still under development. You can see the <a class="underline" href="https://github.com/ecamp/ecamp3/milestone/7">current status</a> on GitHub.`,
```

Comment:
Not sure about that one. The software is under development, but we are in the development phase.

Reply:
I think I would prefer "in the development phase"